src/components-v2/layouts/ListContentLayout/Layout.jsx

```diff
@@ -4,43 +4,44 @@ import styled from 'styled-components'
 export const Wrapper = styled.div`
   display: grid;
   grid-template-columns: 245px auto;
-  grid-template-rows: 62px 500px 25px;
-  min-height: 500px;
+  grid-template-rows: 500px;
+  min-height: 525px;
 
   .background {
     box-shadow: 1px 2px 10px 0 rgba(212, 212, 211, 0.59);
     background-color: white;
   }
 `
-export const Nav = styled.div`
-  grid-column: 1/3;
-  grid-row: 1;
-  margin: 8px 0;
-  padding: 16px 0;
-  box-sizing: border-box;
-  display: flex;
-  justify-content: flex-end;
-`
+// export const Nav = styled.div`
+//   grid-column: 1/3;
+//   grid-row: 1;
+//   margin: 8px 0;
+//   padding: 16px 0;
+//   box-sizing: border-box;
+//   display: flex;
+//   justify-content: flex-end;
+// `
 
 export const Menu = styled.div.attrs(() => ({ className: 'background' }))`
   grid-column: 1;
-  grid-row: 2/4;
   border-right: solid 2px #e8e7e6;
   border-top-left-radius: 8px;
   border-bottom-left-radius: 8px;
+  background-color: white;
```

Comment:
could this become a variable?

Reply:
and I don't recall that our design has plain white color anywhere, where is it used?

Reply:
it's being used in several palces.